recipes/protobuf/all/conandata.yml

```diff
@@ -1,10 +1,10 @@
 sources:
-  "3.19.2":
-    url: "https://github.com/protocolbuffers/protobuf/archive/v3.19.2.tar.gz"
-    sha256: "4dd35e788944b7686aac898f77df4e9a54da0ca694b8801bd6b2a9ffc1b3085e"
```

Comment:
I understand, but this version is still used in 4 recipes.

Reply:
Sure.